projects/plugins/boost/app/modules/cache/Boost_Cache_Settings.php

```diff
@@ -51,6 +51,7 @@ private function init_settings() {
 
 		$lines = file( $this->config_file );
 		if ( count( $lines ) < 4 ) {
+			Logger::debug( 'Invalid config file' );
```

Comment:
Given logging won't be turned on if the config file is invalid, this will never get logged once we're checking the settings before logging.

Reply:
(Same goes for all calls to Logger::debug inside the settings reading code)